terraform/modules/hub/hub_frontend.tf

```diff
@@ -92,62 +88,22 @@ module "frontend_ecs_roles" {
   image_name       = "verify-frontend"
 }
 
-resource "aws_ecs_task_definition" "frontend" {
-  family                = "${var.deployment}-frontend"
-  container_definitions = data.template_file.frontend_task_def.rendered
-  network_mode          = "awsvpc"
-  execution_role_arn    = module.frontend_ecs_roles.execution_role_arn
-}
-
-# This is called frontend_v2 because there was an old frontend service
-# that coexisted at the same time for a while and life is sometimes
-# too short to `terraform state mv`
-resource "aws_ecs_service" "frontend_v2" {
-  name            = "${var.deployment}-frontend-v2"
-  cluster         = aws_ecs_cluster.ingress.id
-  task_definition = aws_ecs_task_definition.frontend.arn
-
-  desired_count                      = var.number_of_apps
-  deployment_minimum_healthy_percent = 50
-  deployment_maximum_percent         = 100
-
-  load_balancer {
-    target_group_arn = aws_lb_target_group.ingress_frontend.arn
-    container_name   = "nginx"
-    container_port   = "8443"
-  }
-
-  network_configuration {
-    subnets = aws_subnet.internal.*.id
-
-    security_groups = [
-      aws_security_group.frontend_task.id,
-      aws_security_group.can_connect_to_container_vpc_endpoint.id,
-    ]
-  }
-
-  service_registries {
-    registry_arn = aws_service_discovery_service.frontend.arn
-    port         = 8443
-  }
-}
-
 resource "aws_ecs_task_definition" "frontend_fargate" {
   family                   = "${var.deployment}-frontend-fargate"
   container_definitions    = data.template_file.frontend_task_def.rendered
   network_mode             = "awsvpc"
   execution_role_arn       = module.frontend_ecs_roles.execution_role_arn
   requires_compatibilities = ["FARGATE"]
-  cpu                      = var.ingress_instance_type == "t3.xlarge" ? 2048 : 1024
-  memory                   = var.ingress_instance_type == "t3.xlarge" ? 16384 : 2048
+  cpu                      = 1024
+  memory                   = 2048
 }
 
 resource "aws_ecs_service" "frontend_fargate" {
   name            = "${var.deployment}-frontend"
   cluster         = aws_ecs_cluster.fargate-ecs-cluster.id
   task_definition = aws_ecs_task_definition.frontend_fargate.arn
 
-  desired_count                      = var.number_of_apps
+  desired_count                      = var.number_of_frontend_apps
```

Comment:
```suggestion
  desired_count                      = var.deployment != "prod" ? var.number_of_frontend_apps : 16
```

Reply:
This change is temporary and can be removed once the pull request is https://github.com/alphagov/verify-infrastructure-config/pull/354 is merged.

Reply:
I would have thought we could just pause the resources in concourse to ensure the changes go out together

Reply:
It is better than pausing the resources in concourse.

Reply:
Id rather just pause and merge both PRs... Rather than have to get another PR merged after

Reply:
Merge the other PR first to reduce the risk if concerned ... That way it'll just error and not deploy to prod

Reply:
If you unpause the jobs in Concourse, will Concourse push the change (changing the number of frontend apps to 2) to production first and then push the next change (increase the number of frontend apps to 16) to production?

Reply:
The process I usually do for this kind of thing is:

* Pause "deploy-hub-staging" job
* Merge both PRs
* Click "refresh" on each of the git resources in concourse (to ensure they have latest version)
* Wait a little bit
* Unpause the "deploy-hub-staging" job
* Look at the build log in concourse to double check both of the git resources are using the expected version
* If not... pause prod and work out why, else all good :+1:

Reply:
Thanks. It makes sense.